cluster-provision/k8s/1.30/k8s_provision.sh

```diff
@@ -148,9 +148,24 @@ patch $cni_manifest_ipv6 $cni_ipv6_diff
 
 cp /tmp/local-volume.yaml /provision/local-volume.yaml
 
-# TODO use config file! this is deprecated
+# Create drop-in config files for kubelet
+# https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/#kubelet-conf-d
```

Comment:
I see in that link that `KUBELET_CONFIG_DROPIN_DIR_ALPHA` has to be defined, but I don't see it being defined here. WDYT?